vscode-extension/src/features/treeView/rangeExplorer/RangeExplorerView.ts

```diff
@@ -44,7 +50,13 @@ export class RangeExplorerView extends NinjaTreeView {
 
     protected override refreshAfterConfigChange(app: ALApp): void {
         const node = this._appRoots.get(app);
-        this._onDidChangeTreeData.fire(node);
+        if (node && node.uri.authority === app.appId)
```

Comment:
Aaaah 😁 Curly braces, please! This especially hurts me when `if (condition) something() else { ... }` Oh my my...

Reply:
Sorry for giving you a heart attack 😆 
Linter to the rescue: 7ba87ccc7cdb1b45172d51296ded88b2e5128580